pkg/services/ngalert/notifier/config.go

```diff
@@ -0,0 +1,77 @@
+package notifier
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/grafana/alerting-api/pkg/api"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+func PersistTemplates(cfg *api.PostableUserConfig, path string) ([]string, bool, error) {
```

Comment:
I think we can delete the old files at this point else it would cause file leak. I wish we could pass string directly instead of files :|